x-pack/plugins/actions/server/usage/actions_telemetry.ts

```diff
@@ -11,7 +11,7 @@ export async function getTotalCount(callCluster: APICaller, kibanaIndex: string)
     scripted_metric: {
       init_script: 'state.types = [:]',
       map_script: `
-        String actionType = doc['action.actionTypeId'].value;
+        String actionType = doc['action.actionTypeId'].value.replace(".", "");
```

Comment:
Is this another case of doing a `s/.//` - I'm sure I reviewed something similar last week.

I'd prefer we come up with some other story for this - like `s/./__` (double underscore).  Not a perfect story, but better than removing them, I think.